apps/aecontract/test/aect_test_utils.erl

```diff
@@ -279,6 +279,74 @@ new_key_pair() ->
     #{ public := PubKey, secret := PrivKey } = enacl:sign_keypair(),
     {PubKey, PrivKey}.
 
+legacy_compile(Vsn, SrcFile) ->
+    Compiler = compiler_cmd(Vsn),
+    OutFile  = tempfile_name("tmp_sophia_", [{ext, ".aeb"}]),
+    Cmd = Compiler ++ " " ++ SrcFile ++ " -o " ++ OutFile,
+    Output = os:cmd(Cmd),
+    try
+        {ok, Bin} = file:read_file(OutFile),
+        aeser_api_encoder:safe_decode(contract_bytearray, Bin)
+    catch _:_ ->
+        {error, <<"Compiler error">>}
+    after
+        cleanup_tempfiles()
+    end.
+
+compiler_cmd(Vsn) ->
+    BaseDir = filename:join([code:priv_dir(aesophia_cli), "bin"]),
+    case Vsn of
+        ?SOPHIA_ROMA    -> filename:join([BaseDir, "v1.4.0", "aesophia_cli"]);
```

Comment:
I see that apps/aehttp/priv/bin/aesophia is not touched by this PR, and this is for test. Ok.

Reply:
No, that should be removed in 3.0.0 (and https://www.pivotaltracker.com/story/show/164597852 should be finished)